File: src/main/java/kr/co/infob/common/database/entity/UserInfo.java
package kr.co.infob.common.database.entity;

import java.util.Date;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter @Setter
@RequiredArgsConstructor
public class UserInfo {

	private String userId;

	private String passwd;

	private String userNm;

	private String cttpc;

	private String email;

	private Date registDttm;

	private String updusrId;

	private Date updtDttm;

}

File: src/main/frontend/app/component/entity/UserInfo.ts
export namespace UserInfo {

    export type User = {
        userId: string;
        passwd: string;
        userNm: string;
        cttpc: string;
        email: string;
        registDttm: Date;
        updusrId: string;
        updtDttm: Date;

        token?: {
            accessToken: string;
            refreshToken: string;
        }
    }
    
}
